src/page-response-impact/BaselinePopulationModal/index.tsx

```diff
@@ -58,12 +57,20 @@ const Subtitle = styled.span`
   }
 `;
 
-const ModalTitle: React.FC<Pick<Props, "numFacilities">> = ({
+const ModalTitle: React.FC<Pick<Props, "numFacilities"> & { page: number }> = ({
```

Comment:
cool, had to look up this TS syntax. Should we explicitly define a prop type for this component for clarity?

Reply:
Sure, I can pull this definition out and assign it to a type